Completion/Unix/Command/_git

```diff
@@ -7179,8 +7194,14 @@ __git_changed-in-index_files () {
 
 (( $+functions[__git_treeish-to-index_files] )) ||
 __git_treeish-to-index_files () {
-  local tree=$1; shift
-  __git_diff-index_files $tree "files different between ${(qq)tree} and the index" treeish-to-index-files "$@"
+  __git_staged_files
+}
```

Comment:
Given that the incumbent code uses `$1` and the new code doesn't, I don't see how this can possibly be correct.